README.md

```diff
@@ -1,6 +1,147 @@
 # GTKWave
+
 GTKWave is a fully featured GTK+ based wave viewer for Unix and Win32 which reads LXT, LXT2, VZT, FST, and GHW files as well as standard Verilog VCD/EVCD files and allows their viewing.
 
-This repository contains two versions of GTKWave:
```

Comment:
Since this will become the main branch, what people read when they clone the repo or they visit it in the browser, I believe it would be desirable a note in this regard. For example:

"This branch contains the gtkwave3-gtk3 version with support for GTK+ 2 and GTK+ 3. See also branches XXXX and YYYYY."

Reply:
That makes sense. I'll include this change in a future PR, because I also want to update the README a bit more. At the moment I want to at least include a screenshot and a link to the PDF documentation in the README. At a later date this could then be updated to point to the new docs website, which I started in #144 and didn't have time to finish and convert to Sphinx yet.